library/src/webapp/editor/ckextraplugins/common-wl/js/folder-listing.js

```diff
@@ -155,6 +156,9 @@ $.fn.folderListing = function(options) {
 
 // automatically bind to data-folder-listing divs
 $(document).ready(function() {
-  $('[data-folder-listing]').folderListing();
+  //Check if 'no-highlight' class is present or not and bind accordingly.
+  if (!($('.no-highlight').length > 0)){
+    $('[data-folder-listing]').folderListing();
+  }
 });
```

Comment:
Let's see if we can extract the calling of this plugin so that it's easy for other people to use